src/messages/FileMessage.vue

```diff
@@ -25,12 +64,39 @@ export default {
     messageColors: {
       type: Object,
       required: true
+    },
+  },
+  methods:{
+    humanReadableFileSize(aSize) {
+      const prettyBytes = require('pretty-bytes');
+      return prettyBytes(aSize);
+    },
+    truncateFilename(filename, maxLength) {
+      if(filename.length > maxLength) {
```

Comment:
maybe we can also avoid this check altogether, that would simplify the code :) A string that is already shorter that the threshold should be left untouched by the smartTruncate package.